.gitignore

```diff
@@ -34,4 +34,5 @@ Extensions/BitBucket/DownloadArtifactBitbucket/bin/*
 Extensions/BitBucket/DownloadArtifactBitbucket/obj/*
 
 ## Nuget 
-nuget.exe
\ No newline at end of file
+nuget.exe
+Extensions/**/DeploymentSDK
```

Comment:
addressed in commit [bae6112](https://github.com/mmikkelson/vsts-rm-extensions/commit/bae6112afe5e49f473b944a6fe99992ccbf0df97)